infra/presubmit.py

```diff
@@ -132,22 +132,32 @@ def check_project_yaml_constants(self):
         continue
       actual_constants = self.data[section]
       for constant in actual_constants:
-        if constant not in allowed_constants:
-          self.print_error('%s (in %s section) is not one of %s', constant,
-                           section, allowed_constants)
+        if isinstance(constant, str):
+          if constant not in allowed_constants:
+            self.error(
+                '{constant} (in {section} section) is not one of {allowed_constants}'
```

Comment:
is not a valid constant ({allowed_constants}).

to match style.

Reply:
oops, forgot this one. Now it's also Done.